### PR TITLE
ADC Configuration introduced

### PR DESCRIPTION
**Pull request by ihor96berizka**
Link to ADC library: https://github.com/espressif/arduino-esp32/blob/master/cores/esp32/esp32-hal-adc.h

### CODE REVIEW

File: init-sensor.h [hunk 1]
```diff
@@ -42,6 +42,9 @@ void initSensorType() {
     case SENSOR_DS18B20:
       initDallasSensor();
       break;
+    case SENSOR_ANALOG_TEST:
```
Comment (shybzzz) [inline init-sensor.h:45]: should we do that only for potentiometer? (SENSOR_ANALOG_TEST is potentiometer)?
or should we do that for GUT800 and INA as well?
or should we do that for GUT800 and INA in another way?

Comment (alexsuch) [inline init-sensor.h:45]: You should init 0dB attenuation also for SENSOR_GUT800 .
Maybe we should set attenuation like configurable parameter for the projects which use ADC as well

Comment (ihor96berizka) [reply init-sensor.h:45]: Do we need to configure attenuation only or rest ADC-related parameters too?

Comment (ihor96berizka) [reply init-sensor.h:45]: I guess that this should be done for GUT800 and INA only.
For potentiometer we need to measure whole range(0-3.3V).

Comment (alexsuch) [reply init-sensor.h:45]: No. Attenuation is the main parameter. But we can reserve code/memory for other possible usage of ADC related parameters: for.ex. ADC resolution

Comment (shybzzz) [reply init-sensor.h:45]: DS also has resolution and a lot of another configurations
It looks like we need to have sensor configurations and those configurations may be different for different sensor types 
So we need to think about some approach how we handle them

**Review by shybzzz — APPROVED** [review]

File: init-sensor.h (hunk 1 above)
Comment (alexsuch) [reply init-sensor.h:45]: @shybzzz In which cases we are using potentiometer solution? What is the purpose of this device?

**Pull request merged**